Libraries/Animated/src/Interpolation.js

```diff
@@ -163,6 +165,16 @@ function interpolate(
   return result;
 }
 
+function colorToRgba(
+  input: string
+): string {
+  var color = tinycolor(input);
+  var {r, g, b, a} = color.toRgb();
```

Comment:
Are we talking about the same thing?
Please respond😀
On Oct 4, 2015 6:47 AM, "Michiel Dral" <notifications@github.com> wrote:

> In Libraries/Animated/src/Interpolation.js
> <https://github.com/facebook/react-native/pull/3177#discussion_r41096637>:
>
> > @@ -163,6 +165,16 @@ function interpolate(
> >    return result;
> >  }
> >
> > +function colorToRgba(
> > +  input: string
> > +): string {
> > +  var color = tinycolor(input);
> > +  var {r, g, b, a} = color.toRgb();
>
> Not safe? Tinycolor default to black, so it won't crash!
> What do you mean by not safe?
> (I'll change it anyway ^^)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react-native/pull/3177/files#r41096637>.
>